### DISCUSSION TITLE
Is it possible to change the log level to WARN?

### DISCUSSION THREAD

**Discussion by dathbe** [mealie-recipes/mealie#3015] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I was just curious if there is a way to set the log level to WARN instead of INFO to declutter the log.  I found this page [here](https://github.com/mealie-recipes/mealie/blob/mealie-next/gunicorn_conf.py), but this file does not seem to be mapped to a persistent docker volume.  Perhaps there's a docker environment variable I can set?
Thanks!

### Please provide a concise description of the feature that would resolve your issue.

Just a question.

### Please consider and list out some caveats or tradeoffs made in your design decision

Just a question.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by cmintey** [comment]
You should be able to set `LOG_LEVEL=warn` (or it might be called `warning`) in your environment

**Comment by dathbe** [comment]
No luck.  ```LOG_LEVEL=warn``` and ```LOG_LEVEL=warning``` throw keyErrors.  ```LOG_LEVEL=WARN``` and ```LOG_LEVEL=WARNING``` will boot, but my docker logs are still showing INFO level logs.

**Reply by cmintey** [reply]
That works for DEBUG, so I'm not sure why it doesn't work for WARN
